packages/destination-actions/src/destinations/facebook-conversions-api/viewContent/index.ts

```diff
@@ -0,0 +1,100 @@
+import { ActionDefinition, IntegrationError } from '@segment/actions-core'
+import { action_source, contents, content_category, content_ids, content_name, content_type, currency, event_id, event_source_url, event_time, value } from '../fb-capi-properties'
+import { user_data_field, hash_user_data } from '../fb-capi-user-data'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { CURRENCY_ISO_CODES, API_VERSION } from '../constants'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'View Content',
+  description: 'Send event when a user views content or a product',
+  defaultSubscription: 'type = "track" and event = "Product Viewed"',
+  fields: {
+    user_data: user_data_field,
+    event_time: { ...event_time, required: true },
+    action_source: { ...action_source, required: true },
+    event_id: event_id,
+    event_source_url: event_source_url,
+    content_ids: { ...content_ids, default: { '@path': '$.properties.product_id' } },
+    content_category: content_category,
+    content_name: content_name,
+    content_type: content_type,
+    contents: { 
+      ...contents,
+      default: [{
+        item_price: {
+          '@path': '$.properties.price'
+        },
+        id: {
+          '@path': '$.product_id'
```

Comment:
I'm not sure what the correct syntax is when we use arrayPath but I think this should be found within properties.

```suggestion
          '@path': '$.properties.product_id'
```